src/views/clusteroverview/overview/components/details-card/utils/models.ts

```diff
@@ -0,0 +1,69 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk';
+
+export const ClusterServiceVersionModel: K8sModel = {
+  kind: 'ClusterServiceVersion',
+  label: 'ClusterServiceVersion',
+  // t('ClusterServiceVersion')
+  labelKey: 'ClusterServiceVersion',
+  labelPlural: 'ClusterServiceVersions',
+  // t('ClusterServiceVersions')
+  labelPluralKey: 'ClusterServiceVersions',
+  apiGroup: 'operators.coreos.com',
+  apiVersion: 'v1alpha1',
+  abbr: 'CSV',
+  namespaced: true,
+  crd: true,
+  plural: 'clusterserviceversions',
+  propagationPolicy: 'Foreground',
+  legacyPluralURL: true,
+};
+
+export const SubscriptionModel: K8sModel = {
```

Comment:
this model already exists in '@kubevirt-ui/kubevirt-api/console/models/SubscriptionModel'

Reply:
Fixed!